src/main/java/me/ryanhamshire/GriefPrevention/UUIDFetcher.java

```diff
@@ -98,6 +99,18 @@ public void call() throws Exception
         //for online mode, call Mojang to resolve the rest
         if (GriefPrevention.instance.getServer().getOnlineMode())
         {
+            Pattern validNamePattern = Pattern.compile("^\\w+$");
+
+            // Don't bother requesting UUIDs for invalid names from Mojang.
+            names.removeIf(name ->
+            {
+                if (name != null && name.length() >= 3 && name.length() <= 16 && validNamePattern.matcher(name).find())
```

Comment:
Is the null check necessary here? It might be better to ensure that no `null` values exist altogether in the `names` object.

Reply:
True enough. It's technically possible, and BigScary's style was to add extra defensive coding everywhere. In the spirit of a minimal diff I didn't add anything outside of online mode.